### ISSUE TITLE
Hyperbahn: Fix rate limiter stats to line up with request volume

### ISSUE THREAD

**Issue by Raynos** [uber/tchannel#1138]
The total rate per service number is wildly different from the inbound request volume number.

We should probably remove all the guages.

**Comment by Raynos** [comment]
cc @ShanniLi

**Comment by ShanniLi** [comment]
Yes, gauges may not be the right stats there. Let me try counter.

**Comment by ShanniLi** [comment]
@Raynos I looked at the dashboard, the stats is correct. You cannot use sum with gauges, because it is just a sampling every one second. If you want to use sum, we need to use counter. In this case, there will be a little perf degradation.

**Comment by Raynos** [comment]
@ShanniLi I want to use sum; we need to use a counter. 

Counters should be more performant then guages.

**Comment by ShanniLi** [comment]
OK, I will make the change
